lms/resources/_js_config.py

```diff
@@ -392,5 +392,6 @@ def _sync_api(self):
                     "context_id": req.params["context_id"],
                     "custom_canvas_course_id": req.params["custom_canvas_course_id"],
                 },
+                "group_info": dict(req.params),
```

Comment:
Should probably reduce this to just the necessary subset of params

Reply:
Are these "group_info" or are they "analytics_info" or something? I feel like group info in general is not the right name.

Reply:
I think `group_info` is fine: the table is one row per group (whether a course group or section group) and it contains the group's `authority_provided_id` (intended to be used to look up the group in the h DB) plus a bunch of "contextual" information about the group: the LTI params the group was last launched with, and a list of instructors who've launched the groups.